pages/search.js

```diff
@@ -28,7 +27,7 @@ import { H1, P } from '../components/Text';
 const SearchInput = styled(StyledInput)`
   &&& {
     border: none;
-    border-bottom: 2px solid ${colors.blue};
+    border-bottom: 2px solid ${props => props.theme.colors.blue[20]};
```

Comment:
```suggestion
    border-bottom: 2px solid ${props => props.theme.colors.primary[500]};
```

Reply:
@Betree Happy to make this change but I don't understand how primary[500] references the previously used blue color, `#3385FF`, which is not in the *lib/theme/colors.js* file.

Reply:
As per my previous comment:
> No need to create a new one, as mentioned in the issue description we should replace them with existing colors from the design system.

It's ok if colors don't match at 100%. We want to use the latest values from our current design system, not the one that was in place when this file was created.